### ISSUE TITLE
Finding duplicate blocks

### ISSUE THREAD

**Issue by nhanvu327** [athensresearch/athens#860]
As a user using the Zettelkasten method for a few years, I see there is a problem that sometimes I forgot I made a note about something, for example, a concept, so I made another one. This is inefficient because the knowledge base is fragmented and hence creating connections between ideas is more difficult. I suggest there should be a function to detect duplication between blocks or pages. The only concern is the performance, I believe this request is easier to implement if this one https://github.com/athensresearch/athens/issues/583 implemented first.

**Comment by nhanvu327** [comment]
do you agree @tangjeff0 ?

**Comment by tangjeff0** [comment]
Yeah this is a really interesting feature. It's sort of like auto-suggesting links based on what you've written - right now most networked notetaking apps only suggest by plain-text match - except it's auto-suggesting block refs.

I could see Concept-Descriptor being helpful to making it happen, but I'm also not sure it's totally necessary.
